JSIL/AST/JSAstVisitor.cs

```diff
@@ -216,6 +233,11 @@ protected class Adapter<TVisitor, TNode>
                 NodeIndex = oldNodeIndex;
                 StatementIndex = oldStatementIndex;
             }
+
+            if (node != null && AfterNodeProcessed != null)
```

Comment:
OK. Probably indicates a bug on my end. Don't mind having the conditional there, though.